### ISSUE TITLE
Bump development status in sub-projects setup.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6696]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#6696|
|Type|enhancement|
|Created|2013-08-23 18:08:46Z|
|Branch|https://github.com/twisted/twisted/tree/dev-status-6696|

The setup.py for conch, mail, words and names (eg. [twisted/names/topfiles/setup.py](twisted/names/topfiles/setup.py)), currently use the "Development Status :: 4 - Beta" [classifier](https://pypi.python.org/pypi?%3Aaction=list_classifiers) but after 10 years it's probably save to bump it to "Development Status :: 5 - Production/Stable".

<details><summary>Searchable metadata</summary>

```
trac-id__6696 6696
type__enhancement enhancement
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_dev_status_6696 branches/dev-status-6696
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1377281326000000 1377281326000000
changetime__1382949386000000 1382949386000000
version__None None
owner__lvh lvh
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="shogun888's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|shogun888 commented|
|-|-|

oops, I misunderstood what was asked for. can i remove my patch file?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

Replying to [shogun888](#note_1):
> oops, I misunderstood what was asked for. can i remove my patch file?
Removed it for you. Please read [TwistedDevelopment#SubmittingaPatch](https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch) on how to submit a patch, thanks.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [40071]) Branching to 'dev-status-6696'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [40072]) bump development status to '5 - Production/Stable' for subprojects that were still marked as beta. refs [#6696](https://github.com/twisted/twisted/issues/6696)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

Forced a [build](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/dev-status-6696).

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_6"></a>|@lvh set owner to @lvh|
|-|-|

Only one buildbot failed with that stupid Windows ENOSPC bug. Otherwise this looks great, so I'll merge it :)

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_7"></a>|@lvh set status to `closed`|
|-|-|

(In [40515]) Merge dev-status-6696: mark mail, conch, words and names as stable.

Author: thijs
Reviewer: lvh
Fixes: [#6696](https://github.com/twisted/twisted/issues/6696)

The packages mail, conch, words and names were still marked as beta. Given their years of development, the many production users and Twisted's stringent compatibility policies, we can almost certainly mark them as production-ready.
